src/transit/tests.py

```diff
@@ -102,6 +122,13 @@ def generate_tests(base_url):
             ),
             callback=no_walking_routes_in_boarding_soon,
         ),
+        Test(
+            name="api/v1/search contains googlePlaces and busStops",
+            request=Request(
+                method="POST", url=base_url + "api/v2/search/", payload={"query": "st"}
```

Comment:
separate onto individual lines

Reply:
the linter didn't get mad at this, not sure if i should do that? @young-k